commercial/app/controllers/nonRefreshableLineItemsController.scala

```diff
@@ -16,8 +16,14 @@ class nonRefreshableLineItemsController(val controllerComponents: ControllerComp
     Action { implicit request =>
       val nonRefreshableLineItems: Seq[Long] = DfpAgent.nonRefreshableLineItemIds()
       val json = Json.toJson(nonRefreshableLineItems)
-      Cached(15.minutes) {
-        JsonComponent(json)
-      }
+
+      Cors(
```

Comment:
I wasn't sure whether it was worth limiting this to the CODE env only, since we don't need this file to have CORS added in the PROD env. I decided it wasn't worth adding the complexity

Reply:
I think this is fine!